lib/Sub/Override.pm

```diff
@@ -187,6 +121,62 @@ sub DESTROY {
     }
 }
 
+sub _normalize_sub_name {
```

Comment:
A tiny nit (my fault) that's bugged me for a long time. This should probably be renamed something like `_get_fully_qualified_sub_name` or something like that.

Reply:
Good idea. I noticed my wrap routine has a mis-named variable. I may take a look at the variable names in general and fix anything else I find, sometime.